nixos/modules/services/mail/rainloop.nix

```diff
@@ -0,0 +1,110 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.rainloop;
+  instance = name: { package, php-package, domain, ... }: let
+    home = "/var/lib/rainloop/${name}";
+    long-name = "rainloop-${name}";
+  in {
+    user.name = long-name;
+    user.value = {
+      isSystemUser = true;
+      description = "Rainloop user for ${name}";
+      createHome = true;
+      inherit home;
+    };
+
+    pool.name = long-name;
+    pool.value = {
+      user = long-name;
+      group = "nogroup";
+      settings = {
+        "env[RAINLOOP_DATA_DIR]" = home;
+        "listen.owner" = config.services.nginx.user;
+        "pm" = "dynamic";
+        "pm.max_children" = 32;
+        "pm.max_requests" = 500;
+        "pm.start_servers" = 2;
+        "pm.min_spare_servers" = 2;
+        "pm.max_spare_servers" = 5;
+        "php_admin_value[error_log]" = "stderr";
+        "php_admin_flag[log_errors]" = true;
+        "catch_workers_output" = true;
+      };
+      phpEnv."PATH" = lib.makeBinPath [ php-package ];
+      phpOptions = ''
+        upload_max_filesize = 40m
+        post_max_size = 49M
+      '';
+    };
+
+    vhost.name = domain;
+    vhost.value = {
+      extraConfig = ''client_max_body_size 50m;'';
```

Comment:
```suggestion
      extraConfig = ''
        client_max_body_size 50m;
      '';
```

Reply:
What do the extra linebreaks do?

Reply:
Nothing, but they make it more readable, IMO and future diffs look better.